docs/resources/Channel.md

```diff
@@ -114,6 +114,9 @@ Bots can post or publish messages in this type of channel if they have the prope
 
 ###### Example Guild Voice Channel
 
+> warn
+> Text in Voice: Guild Voice channels contain chats, are considered text-based channels, but do not support threads and pinned messages.
```

Comment:
```suggestion
> Guild Voice channels are considered text-based channels, but do not support threads and pinned messages.
```

Also, does this need to be a warning?

Reply:
Uhmm, I don't really know, because with a "text-based" channel I refer to a channel that has similarities with Text Channels, but I need to "specify" things that are in a common Guild Text Channels, not supported on a Guild Voice Channel, because we are on a Guild context. What do you think?

Reply:
I don’t think we need a warning box to explain standard app functionality; the other channel types don’t have this

Reply:
So, it will be better if I remove all of this?

```markdown
> warn
> Text in Voice: Guild Voice channels contain chats, are considered text-based channels, but do not support threads and pinned messages.
```

Reply:
I will just "revert" all I made and update the example/payload of the Guild Voice Channel.